mon-pix/app/styles/pages/_sign-form.scss

```
@@ -184,7 +178,7 @@
 
   &__forgotten-password-link {
     align-self: flex-end;
-    margin: 5px 0 20px;
+    margin-top: 5px;
```

Serait-il possible d'en profiter pour utiliser un token du DS pour ces valeurs ?

---

Il n'y a pas de token dont la valeur correspond à 5px

---

Dans ce cas, plutôt utiliser `--pix-spacing-1x` qui correspond à 4px ? 
```suggestion
    margin-top: var(--pix-spacing-1x);
```